app/mailers/user_mailer.rb

```diff
@@ -33,6 +33,7 @@ def new_hosts_digest(visit, visitor, host_data)
     deliver_message(
       from: default_sender_address,
       to: @visitor.email,
+      'h:Reply-To': 'DO-NOT-REPLY@berniebnb.com',
```

Comment:
I think for string keys you're going to need `'h:Reply-To' => DO-NOT-REPLY@berniebnb.com`